templates/repo/commits_table.tmpl

```diff
@@ -60,7 +60,7 @@
 							</a>
 						</td>
 						<td class="message collapsing">
-							<span class="has-emoji{{if gt .ParentCount 1}} grey text{{end}}">{{RenderCommitMessage .Summary $.RepoLink $.Repository.ComposeMetas}}</span>
+							<span class="has-emoji{{if gt .ParentCount 1}} grey text{{end}}" title={{.Message}}>{{RenderCommitMessage .Summary $.RepoLink $.Repository.ComposeMetas}}</span>
```

Comment:
Quotes seems to be missing

Reply:
I added the quotes and title in `page.tmpl`